include/rocksdb/db.h

```diff
@@ -621,6 +621,10 @@ class DB {
     // "rocksdb.block-cache-pinned-usage" - returns the memory size for the
     //      entries being pinned.
     static const std::string kBlockCachePinnedUsage;
+
+    // "rocksdb.options-statistics" - returns multi-line string
```

Comment:
This confuses me that the property name kStatistics = "rocksdb.options-statistics".